docs/common_tasks.md

```diff
@@ -41,3 +41,11 @@ To add a new repository, the following steps are needed:
 
 For more information on adding a new GitHub repository, see
 [Adding a GitHub repository](submitting_data.md#adding-a-github-repository).
+
+## Debugging Tools
+
+You can use the Python Debugger ([pdb](https://docs.python.org/3.7/library/pdb.html)) in a docker container.
+After starting a local treeherder instance using [docker-compose](installation.md#server-and-full-stack-development),
+in a separate shell type `docker attach backend`. Then set a breakpoint in your file using either `import pdb; pdb.set_trace()`
```

Comment:
That's really simple. Thank you for finding out this trick!

Reply:
I'm glad you mentioned pdb last week when I was asking you about venv and vscode debugging. :)